source/Weapon.cpp

```diff
@@ -123,6 +123,10 @@ void Weapon::LoadWeapon(const DataNode &node)
 					submunitions.back().facing = Angle(grand.Value(1));
 				else if((grand.Size() >= 3) && (grand.Token(0) == "offset"))
 					submunitions.back().offset = Point(grand.Value(1), grand.Value(2));
+				else if(grand.Size() >= 2 && grand.Token(0) == "spawns on natural death" && grand.Value(1) <= 0)
+					submunitions.back().spawnOnNaturalDeath = false;
+				else if(grand.Size() >= 2 && grand.Token(0) == "spawns on anti-missile death" && grand.Value(1) > 0)
```

Comment:
these are quite a mouthful I would prefer a shorter name

Reply:
Yes I know. Would "natural" and "anti-missile" be better?